web/src/components/meetings/MeetingListItem.vue

```diff
@@ -82,6 +85,12 @@ export default {
         this.suggestions = countData.suggestions;
         this.progression = countData.progression;
       }
+    },
+    formatDate(date, formatting) {
```

Comment:
Näitä hyvä miettiä kannattaako nää siirtää sinne dateutils tms alle

Reply:
jesh

Reply:
Uudelleennimesin vanhan päivämäärähelpperin kattavammaks ja laitoin sinne nää omatekemät funktiot. Hyvä huomio!